packages/nextjs/components/scaffold-eth/Contract/utilsContract.tsx

```diff
@@ -82,29 +84,31 @@ const getContractVariablesAndNoParamsReadMethods = (
  * which has corresponding input field for param type and button to read
  */
 const getContractReadOnlyMethodsWithParams = (
-  contract: Contract,
+  contract: Contract | null,
   contractMethodsAndVariables: FunctionFragment[],
 ): { loaded: boolean; methods: (JSX.Element | null)[] } => {
-  return {
-    loaded: true,
-    methods: contractMethodsAndVariables
-      .map((fn, index) => {
-        const isQueryableWithParams =
-          (fn.stateMutability === "view" || fn.stateMutability === "pure") && fn.inputs.length > 0;
-        if (isQueryableWithParams) {
-          return (
-            // FFR -> FunctionFormRead
-            <ReadOnlyFunctionForm
-              key={`FFR_${fn.name}_${index}`}
-              functionFragment={fn}
-              contractAddress={contract.address}
-            />
-          );
-        }
-        return null;
-      })
-      .filter(n => n),
-  };
+  return contract
+    ? {
+        loaded: true,
+        methods: contractMethodsAndVariables
+          .map((fn, index) => {
+            const isQueryableWithParams =
+              (fn.stateMutability === "view" || fn.stateMutability === "pure") && fn.inputs.length > 0;
+            if (isQueryableWithParams) {
+              return (
+                // FFR -> FunctionFormRead
+                <ReadOnlyFunctionForm
+                  key={`FFR_${fn.name}_${index}`}
```

Comment:
and here

Reply:
check https://github.com/scaffold-eth/se-2/pull/96#discussion_r1080773581